frontend/src/App.js

```diff
@@ -4,15 +4,21 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import './App.css';
-
+import { L10nState } from 'core/l10n';
+import { LocaleState } from 'core/locales';
 import { Lightbox } from 'core/lightbox';
 import * as locales from 'core/locales';
 import { Navigation } from 'core/navigation';
 import { UserAutoUpdater } from 'core/user';
 import { EntitiesList } from 'modules/entitieslist';
 import { EntityDetails } from 'modules/entitydetails';
 import { SearchBox } from 'modules/search';
+import { WaveLoader } from './core/loaders';
```

Comment:
This should be an absolute import, without `./` thus, and should be placed by alphabetical order of the "from". So, right after `import { Lightbox } from 'core/lightbox';` in this case.